src/CalculatorUnitTests/CalculatorUnitTests.vcxproj

```diff
@@ -132,7 +131,8 @@
   </ImportGroup>
   <PropertyGroup Label="UserMacros">
     <PackageCertificateKeyFile>CalculatorUnitTests_TemporaryKey.pfx</PackageCertificateKeyFile>
-    <PackageCertificateThumbprint>BE776C83699DBCE4969DE9EE730AEAF66C5D5182</PackageCertificateThumbprint>
+    <PackageCertificateThumbprint>479B71D75C958053AEE4A3D02C623056D5C932C4</PackageCertificateThumbprint>
```

Comment:
Do we actually need to call this out explicitly in addition to the pfx entry above?